// @certusone/wormhole-sdk/lib/esm/ethers-contracts/factories/ERC165Storage__factory.js
/* Autogenerated file. Do not edit manually. */
/* tslint:disable */
/* eslint-disable */
import { Contract, utils } from "ethers";
var _abi = [
    {
        inputs: [
            {
                internalType: "bytes4",
                name: "interfaceId",
                type: "bytes4",
            },
        ],
        name: "supportsInterface",
        outputs: [
            {
                internalType: "bool",
                name: "",
                type: "bool",
            },
        ],
        stateMutability: "view",
        type: "function",
    },
];
var ERC165Storage__factory = /** @class */ (function () {
    function ERC165Storage__factory() {
    }
    ERC165Storage__factory.createInterface = function () {
        return new utils.Interface(_abi);
    };
    ERC165Storage__factory.connect = function (address, signerOrProvider) {
        return new Contract(address, _abi, signerOrProvider);
    };
    ERC165Storage__factory.abi = _abi;
    return ERC165Storage__factory;
}());
export { ERC165Storage__factory };
