// @certusone/wormhole-sdk/lib/esm/ethers-contracts/factories/ERC2771Context__factory.js
/* Autogenerated file. Do not edit manually. */
/* tslint:disable */
/* eslint-disable */
import { Contract, utils } from "ethers";
var _abi = [
    {
        inputs: [
            {
                internalType: "address",
                name: "forwarder",
                type: "address",
            },
        ],
        name: "isTrustedForwarder",
        outputs: [
            {
                internalType: "bool",
                name: "",
                type: "bool",
            },
        ],
        stateMutability: "view",
        type: "function",
    },
];
var ERC2771Context__factory = /** @class */ (function () {
    function ERC2771Context__factory() {
    }
    ERC2771Context__factory.createInterface = function () {
        return new utils.Interface(_abi);
    };
    ERC2771Context__factory.connect = function (address, signerOrProvider) {
        return new Contract(address, _abi, signerOrProvider);
    };
    ERC2771Context__factory.abi = _abi;
    return ERC2771Context__factory;
}());
export { ERC2771Context__factory };
