// @certusone/wormhole-sdk/lib/esm/ethers-contracts/factories/IERC1820Implementer__factory.js
/* Autogenerated file. Do not edit manually. */
/* tslint:disable */
/* eslint-disable */
import { Contract, utils } from "ethers";
var _abi = [
    {
        inputs: [
            {
                internalType: "bytes32",
                name: "interfaceHash",
                type: "bytes32",
            },
            {
                internalType: "address",
                name: "account",
                type: "address",
            },
        ],
        name: "canImplementInterfaceForAddress",
        outputs: [
            {
                internalType: "bytes32",
                name: "",
                type: "bytes32",
            },
        ],
        stateMutability: "view",
        type: "function",
    },
];
var IERC1820Implementer__factory = /** @class */ (function () {
    function IERC1820Implementer__factory() {
    }
    IERC1820Implementer__factory.createInterface = function () {
        return new utils.Interface(_abi);
    };
    IERC1820Implementer__factory.connect = function (address, signerOrProvider) {
        return new Contract(address, _abi, signerOrProvider);
    };
    IERC1820Implementer__factory.abi = _abi;
    return IERC1820Implementer__factory;
}());
export { IERC1820Implementer__factory };
