// @certusone/wormhole-sdk/lib/esm/ethers-contracts/factories/IERC777Sender__factory.js
/* Autogenerated file. Do not edit manually. */
/* tslint:disable */
/* eslint-disable */
import { Contract, utils } from "ethers";
var _abi = [
    {
        inputs: [
            {
                internalType: "address",
                name: "operator",
                type: "address",
            },
            {
                internalType: "address",
                name: "from",
                type: "address",
            },
            {
                internalType: "address",
                name: "to",
                type: "address",
            },
            {
                internalType: "uint256",
                name: "amount",
                type: "uint256",
            },
            {
                internalType: "bytes",
                name: "userData",
                type: "bytes",
            },
            {
                internalType: "bytes",
                name: "operatorData",
                type: "bytes",
            },
        ],
        name: "tokensToSend",
        outputs: [],
        stateMutability: "nonpayable",
        type: "function",
    },
];
var IERC777Sender__factory = /** @class */ (function () {
    function IERC777Sender__factory() {
    }
    IERC777Sender__factory.createInterface = function () {
        return new utils.Interface(_abi);
    };
    IERC777Sender__factory.connect = function (address, signerOrProvider) {
        return new Contract(address, _abi, signerOrProvider);
    };
    IERC777Sender__factory.abi = _abi;
    return IERC777Sender__factory;
}());
export { IERC777Sender__factory };
