// @certusone/wormhole-sdk/lib/esm/ethers-contracts/factories/IERC1363Spender__factory.js
/* Autogenerated file. Do not edit manually. */
/* tslint:disable */
/* eslint-disable */
import { Contract, utils } from "ethers";
var _abi = [
    {
        inputs: [
            {
                internalType: "address",
                name: "owner",
                type: "address",
            },
            {
                internalType: "uint256",
                name: "value",
                type: "uint256",
            },
            {
                internalType: "bytes",
                name: "data",
                type: "bytes",
            },
        ],
        name: "onApprovalReceived",
        outputs: [
            {
                internalType: "bytes4",
                name: "",
                type: "bytes4",
            },
        ],
        stateMutability: "nonpayable",
        type: "function",
    },
];
var IERC1363Spender__factory = /** @class */ (function () {
    function IERC1363Spender__factory() {
    }
    IERC1363Spender__factory.createInterface = function () {
        return new utils.Interface(_abi);
    };
    IERC1363Spender__factory.connect = function (address, signerOrProvider) {
        return new Contract(address, _abi, signerOrProvider);
    };
    IERC1363Spender__factory.abi = _abi;
    return IERC1363Spender__factory;
}());
export { IERC1363Spender__factory };
