// @certusone/wormhole-sdk/lib/esm/ethers-contracts/factories/PullPayment__factory.js
/* Autogenerated file. Do not edit manually. */
/* tslint:disable */
/* eslint-disable */
import { Contract, utils } from "ethers";
var _abi = [
    {
        inputs: [
            {
                internalType: "address",
                name: "dest",
                type: "address",
            },
        ],
        name: "payments",
        outputs: [
            {
                internalType: "uint256",
                name: "",
                type: "uint256",
            },
        ],
        stateMutability: "view",
        type: "function",
    },
    {
        inputs: [
            {
                internalType: "address payable",
                name: "payee",
                type: "address",
            },
        ],
        name: "withdrawPayments",
        outputs: [],
        stateMutability: "nonpayable",
        type: "function",
    },
];
var PullPayment__factory = /** @class */ (function () {
    function PullPayment__factory() {
    }
    PullPayment__factory.createInterface = function () {
        return new utils.Interface(_abi);
    };
    PullPayment__factory.connect = function (address, signerOrProvider) {
        return new Contract(address, _abi, signerOrProvider);
    };
    PullPayment__factory.abi = _abi;
    return PullPayment__factory;
}());
export { PullPayment__factory };
